middleware/validateUpdateEmployeeData.js

```diff
@@ -0,0 +1,44 @@
+function validateUpdateEmployeeData(req, res, next) {
+
+    const { eid, name, age, gender, skills, designation } = req.body;
+    const errors = [];
+
+    if (eid && typeof eid !== 'number') {
```

Comment:
I feel update and add condition logics are almost similar, am I not seeing something
Is there something that differs or can we use same validation file for both

Reply:
Got it!
I misread the code